docs/content/cli.md

```diff
@@ -250,10 +250,10 @@ Global Flags:
 
 ### Bundle List
 
-This command is available both as `porter bundle list` and `porter bundles list`.
+This command is available both as `porter instance list` and `porter bundles list`.
```

Comment:
`s/bundles/instances/`, right?

Reply:
Oops, find/replace strikes again. Fixing. I have next up on my list to autogenerate this page using cobra so I'm not too worried about what's on there though.